cpp/include/cpu_operations.h

```diff
@@ -51,6 +51,20 @@ class CpuOperations {
     // Matrix-matrix multiplication
     return a * b;
   }
+  /// This is a function that adds each element in the matrix to a scalar and
+  /// returns the resulting matrix.
+  ///
+  /// \param a
+  /// Input Matrix
+  /// \param b
+  /// Input Scalar
+  ///
+  /// \return
+  /// This function returns a matrix that is the resulatant of adding the
+  /// input matrix and scalar.
+  ///
+  /// \sa
+  /// \ref Add
```

Comment:
ref should be used for other functions that are related to this one. If I click Add in the webpage, it actually just goes to the same function Add. Is there some way to actually ref the Add function that takes two Matrices here?